python/grass/jupyter/display.py

```diff
@@ -53,25 +53,40 @@ def run(self, module, **kwargs):
         else:
             raise ValueError("Module must begin with letter 'd'.")
 
-    def __getattr__(self, attr, **kwargs):
+    def __getattr__(self, name, **kwargs):
         """Parse attribute to GRASS display module. Attribute should be in
-        the form 'd_{module}'."""
+        the form 'd_{module}'.
 
-        # First check to make sure format is correct
-        if attr[:2] != "d_":
-            print(attr[:2])
-            raise ValueError("Module must begin with 'd_'.")
-        # Now reformat string
-        grass_module = f"d.{attr[2:]}"
+        :param str name: display module in form 'd_{module}'
+        :param kwargs: arguments to be passed to display module
+
+        :return: output of display module
+
+        ----- Example -----
```

Comment:
See how Sphinx recommends formatting examples. I think introducing the example with a sentence is probably enough. See what these docstrings might be turned into:
https://grass.osgeo.org/grass80/manuals/libpython/pygrass.modules.interface.html?highlight=module#module-pygrass.modules.interface.module (and the corresponding source code in grass/pygrass)